packages/flutter/lib/src/material/bottom_sheet.dart

```diff
@@ -493,7 +493,7 @@ PersistentBottomSheetController<T> showBottomSheet<T>({
   return Scaffold.of(context).showBottomSheet<T>(
     builder,
     backgroundColor: backgroundColor,
-    elevation: elevation,
+    elevation: Theme.of(context).bottomSheetTheme.persistentElevation ?? elevation,
```

Comment:
I think this should be the other way around, if the user passes an `elevation` to the function, it should be used instead of the `bottomSheetTheme`s elevation. 


```suggestion
    elevation: elevation ?? Theme.of(context).bottomSheetTheme.persistentElevation,
```

Reply:
Done

Reply:
Agree with this